Create Well-Known-Specification.md (#121)
GoogleChrome/related-website-sets: Well-Known-Specification.md

# The "related-website-set.json" Well-Known Resource Identifier

> This page describes the use and content of URIs having the path 

Reviewer: ```suggestion
This page describes the use and content of URIs with the path 
```

Nit: "having" feels like odd word choice to me.

`/.well-known/related-website-set.json`.
## Use

> A page may be hosted at the URI having the path of 
> `/.well-known/related-website-set.json` in order to demonstrate
> administrative access to the site when submitted as an entry in a Related 
> Website Site (RWS).  

Reviewer: Please use active voice to make this clearer:

```suggestion
A site may demonstrate intentional opt-in to participating in a Related Website Set (RWS) by hosting a specific JSON file at the URI with a path of
`/.well-known/related-website-set.json`.
```

Author: Done

> Website Site (RWS).  

Reviewer: ```suggestion
Website Site (RWS).
```

## Content

> The `/.well-known/related-website-set.json` file for the set primary of an RWS 

Reviewer: Nit: trailing whitespace

```suggestion
The `/.well-known/related-website-set.json` file for the set primary of an RWS
```

must follow the schema specified below:
```json
{
  "type": "object",
  "properties": {
    "ccTLDs": {
      "type": "object",
      "additionalProperties": {
        "type": "array",
        "items": {
          "type": "string"
        }
      }
    },
    "primary": { "type": "string" },
    "associatedSites": {
      "type": "array",
      "items": {
        "type": "string"
      }
    },
    "serviceSites": {
      "type": "array",
      "items": {
        "type": "string"
      }
    },
    "rationaleBySite": {
      "type": "object",
      "additionalProperties": {
        "type": "string"
      }
    }
  },
  "required": ["primary"],
  "dependentRequired": {
    "associatedSites": ["An explanation of how you clearly present the 
    affiliation across domains to users and why users would expect your 
    domains to be affiliated"],
    "serviceSites": ["An explanation of how each domain in this subset 
    supports functionality or security needs."]
  }
}
```

> The `/.well-known/related-website-set.json` file for set members of an RWS 

Reviewer: ```suggestion
The `/.well-known/related-website-set.json` file for set members of an RWS
```

> The `/.well-known/related-website-set.json` file for set members of an RWS 

Reviewer: ```suggestion
The `/.well-known/related-website-set.json` file for non-primary members of an RWS 
```

"set members of an RWS" feels redundant, and I think we also need to specify that we're not talking about the primary sites.
